pages/health-care/my-health-account-validation.md

```diff
@@ -1,5 +0,0 @@
----
-title: My Health Account Validation
-layout: page-react.html
-entryname: my-health-account-validation
----
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-e1a64cfe3be04efcdf8cba6a8cb84477R5.